packages/wallets/src/components/Base/InlineMessage/__tests__/InlineMessage.spec.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import useDevice from '../../../../hooks/useDevice';
+import InlineMessage from '../InlineMessage';
+
+jest.mock('../../../../hooks/useDevice', () => ({
+    __esModule: true,
+    default: jest.fn(() => ({
+        isMobile: false,
+    })),
+}));
+
+describe('InlineMessage', () => {
+    const inLineMessageEl = () => screen.getByTestId('dt_inline_message');
+    const inLineMessageIconEl = () => screen.getByTestId('dt_inline_message_icon');
```

Comment:
minor - thats a function, so it should have name "getInlineMessageIconEl"
if its a variable with element assigned, then it should have name "inlineMessageIconEl"

Reply:
Updated.